queue_job_batch/README.rst

```diff
@@ -38,15 +38,13 @@ Example:
 .. code:: python
 
    from odoo import models, fields, api
-   from odoo.addons.queue_job.job import job
+
 
    class MyModel(models.Model):
-      _name = 'my.model'
+       _name = 'my.model'
```

Comment:
There will be no documentation problem, but you can probably remove falsy space?